### ISSUE TITLE
Precompilation Warning: `Progress(n::Integer, dt::Real, desc::AbstractString = "Progress: ", barlen = nothing, color::Symbol = :green, output::IO = stderr; offset::Integer = 0)` is deprecated

### ISSUE THREAD

**Issue by prbzrg** [tlnagy/TiffImages.jl#113]
```julia
┌ TiffImages [731e570b-9d59-4bfa-96dc-6df516fadf69]
│  ┌ Warning: `Progress(n::Integer, dt::Real, desc::AbstractString = "Progress: ", barlen = nothing, color::Symbol = :green, output::IO = stderr; offset::Integer = 0)` is deprecated, use `Progress(n; dt = dt, desc = desc, barlen = barlen, color = color, output = output, offset = offset)` instead.
│  │   caller = ip:0x0
│  └ @ Core :-1
└
```

**Comment by prbzrg** [comment]
In CI:
https://github.com/tlnagy/TiffImages.jl/actions/runs/6268716116/job/17024093915#step:6:246

**Comment by tlnagy** [comment]
Thanks for the report! This should be fixed in #114
